data/gettingstarted.html

```diff
@@ -1233,6 +1237,46 @@ <h3>&zwnj;Desktop shortcuts&zwnj;</h3>
                 </td>
                 <td>&zwnj;Open Applications menu&zwnj;</td>
               </tr>
+              <tr>
+                <td>
+                  <b class="btn btn-xs disabled">&zwnj;Shift&zwnj;</b>
+                  <i class="material-icons">&#xE145;</i>
+                  <b class="btn btn-xs disabled">&zwnj;Shift&zwnj;</b>
+                  <i class="material-icons">&#xE145;</i>
+                  <b class="btn btn-xs disabled">Print</b>
+                </td>
+                 <td>&zwnj;Copy a screenshot of an area to clipboard&zwnj;</td>
+              </tr>
+              <tr>
+                <td>
+                  <b class="btn btn-xs disabled">&zwnj;Shift&zwnj;</b>
+                  <i class="material-icons">&#xE145;</i>
+                  <b class="btn btn-xs disabled">Print</b>
+                </td>
+                <td>&zwnj;Save a screenshot of an area to Pictures&zwnj;</td>
```

Comment:
"Pictures folder" would be easier to understand I would suggest

Reply:
Added $HOME/Pictures directory